CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 ### AppCenter
 
+#### UWP/WPF/WinForms
+
+* **[Fix]** Fix sending remaining pending logs after sending 3 concurrent HTTP requests.
```

Comment:
This sounds like it fixed not to send remaining logs after it sends 3 concurrent HTTP requests. I think the fix is to send pending logs when it hits the max limit of concurrent calls.

Reply:
I don't see a negation being used here. Feel free to improve the wording on a PR.